.github/CODEOWNERS

```diff
@@ -7,3 +7,4 @@
 
 # GCB configuration.
 cloudbuild-vm.yaml  @wgrzelak @khajduczenia @huyhg
+/docker/ @armandomiani @aav66 @farajn9 @kalinichenka @ovk6 @vcanaa @huyhg
```

Comment:
Is this possible to use teams here?

Reply:
Yes, we can create a team for them.

Reply:
They already have a team @GoogleCloudPlatform/c2d-contributors